test/silicon_runners.jl

```diff
@@ -7,7 +7,7 @@ include("testcases.jl")
 # TODO There is a lot of code duplication in this file ... once we have the ABINIT reference
 #      stuff in place, this should be refactored.
 
-function run_silicon_redHF(;Ecut=5, test_tol=1e-6, n_ignored=0, grid_size=15, scf_tol=1e-6)
+function run_silicon_redHF(T; Ecut=5, test_tol=1e-6, n_ignored=0, grid_size=15, scf_tol=1e-6)
```

Comment:
`T=Float64`?

Reply:
But then also at other places like in `run_silicon_lda` and so on ...
so far we explicitly specify the type everywhere, which I think is actually what you want.